### PR TITLE
Demo fix

### PR DESCRIPTION
**Pull request by SooryaKumaranC-tw**

### CODE REVIEW

File: openmrs/migrations/addLabRoleWithPrivilege.xml [hunk 1]
```diff
@@ -5,14 +5,15 @@
...
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
         http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
-
-    <changeSet id="Lab-Role-20220421" author="Priyanka, Aman">
+  
+    <changeSet id="Lab-Role-20220421-1" author="Priyanka, Aman">
         <preConditions onFail="CONTINUE">
             <sqlCheck expectedResult="0">SELECT COUNT(*) FROM role where role = 'Lab'</sqlCheck>
-            <sqlCheck expectedResult="1">SELECT COUNT(*) FROM privilege where privilege = 'app:lab-lite'</sqlCheck>
+            <sqlCheck expectedResult="0">SELECT COUNT(*) FROM privilege where privilege = 'app:lab-lite'</sqlCheck>
```
Comment (N0-man) [inline openmrs/migrations/addLabRoleWithPrivilege.xml]: this is not correct... bahmni core is adding the privilege so the expectedResults should be 1

Comment (SooryaKumaranC-tw) [reply openmrs/migrations/addLabRoleWithPrivilege.xml]: Bahmni core will run only after this migration executes.

File: README.md [hunk 2]
```diff
@@ -6,25 +6,16 @@ This repo represents configuration for India Clinics.
 
 ### CI
 
-Push to main would trigger 2 yaml files.
-Common things done: Basically gha would SSH into the
+Push to main would trigger gha. It does SSH into the
 ec2 using the pem from repository secrets and perform git clone where the
 clinic_config directory is volume mounted on bahmni-web and openmrs containers
-as default_config
+as default_config. Also a openmrs is restarted by docker.
```
Comment (gsluthra) [inline README.md]: Isn't it mounted as bahmni_config?

Comment (SooryaKumaranC-tw) [reply README.md]: Noted. Will update it.

**Pull request merged**